clients/src/main/java/org/apache/kafka/common/requests/ListOffsetResponse.java

```diff
@@ -136,8 +136,7 @@
 
     public static final class PartitionData {
         public final Errors error;
-        // The offsets list is only used in ListOffsetResponse v0.
-        @Deprecated
+        // Note, the offsets list is only used in ListOffsetResponse v0.
```

Comment:
Removed the `@Deprecated` annotation since both the Kafka broker and the Kafka client need to support old version of the RPC.